.github/dependabot.yml

```diff
@@ -9,11 +9,8 @@ updates:
       interval: "weekly"
 
   - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "weekly"
-
-  - package-ecosystem: "github-actions"
-    directory: "/.github/actions/check-ext-build"
+    directories:
+      - "/"
+      - "/.github/actions/*"
```

Comment:
Does that really work? I'm not sure the glob pattern is supported here, I've seen the following in other repos:
```
    - "/.github/actions/dependabot-check"
    - "/.github/actions/check-ext-build"
```